test/entrypoint/srv_ddl.lua

```diff
@@ -110,11 +111,50 @@ package.preload['customers-storage'] = function()
                 }
             }
 
+            -- DDL module doesn't support map and array types and JSON path in
+            -- sharding key, see https://github.com/tarantool/ddl/issues/81,
+            -- so we create space with jsonpath index manually.
+            local function create_space_with_jsonpath()
+                local customers_jsonpath_key = box.schema.space.create('customers_jsonpath_key', {
+                    format = {
+                        {name = 'id', is_nullable = false, type = 'map'},
+                        {name = 'bucket_id', is_nullable = false, type = 'unsigned'},
+                        {name = 'name', is_nullable = false, type = 'string'},
+                        {name = 'age', is_nullable = false, type = 'number'},
+                        {name = 'data', is_nullable = false, type = 'map'},
+                    },
+                    if_not_exists = true,
+                    engine = engine,
+                })
+                customers_jsonpath_key:create_index('pk', {
+                    parts = {
+                        {1, 'unsigned', path = 'customer_id.unsigned'},
+                        {5, 'unsigned', path = 'customer.weight'},
+                    },
+                    if_not_exists = true,
+                })
+                customers_jsonpath_key:create_index('bucket_id', {
+                    parts = { 'bucket_id' },
+                    unique = false,
+                    if_not_exists = true,
+                })
+                box.space['_ddl_sharding_key']:insert({
+                        'customers_jsonpath_key',
+                        {'name'}
+                })
```

Comment:
The idea of the commit is to verify JSON path as sharding key definition, correct? Hold a correct API and behaviour, but disable it for now, right? But this definition does not use JSON path.

BTW, 'test: add tests for JSON path' does not show any relation to sharding keys. Support of JSON path updates and indexes was added in PR #150 and PR #158.

Please, leave several words about idea of the commit.

Reply:
> The idea of the commit is to verify JSON path as sharding key definition, correct? Hold a correct API and behaviour, but disable it for now, right? But this definition does not use JSON path.

True, it is a part of #219. However, we decided to postpone jsonpath as sharding key definition support. So I moved this to the branch for #219 - `ligurio/gh-219-jsonpath-sharding-key`.